tests/admin/test_admin_users_endpoints.py

```diff
@@ -251,6 +252,18 @@ def test_get_user_noauth(client, db_session):
     r = client.get("/admin/user")
     assert r.status_code == 401
 
+# GET /list_policies test
+
+def test_list_policies(client, admin_user, encoded_admin_jwt):
+    mock_list = conftest.mock_arborist_requests(("GET", "arborist/policies"))
```

Comment:
@microbe73 how did you run this? I ran "pytest tests/admin/test_admin_users_endpoints.py" and this method returned an error. Also when using pytest fixtures you add them to the method parameters so for this method it would be def test_list_policies(mock_arborist_requests, client, admin_user, encoded_admin_jwt): then you would be able to call the mock_arborist_requests inside the test method